### PR TITLE
Adds pendingRetry to error type

### PR DESCRIPTION
**Pull request by TrekSoft**
## Description of the change

Wait for Link v2.2 to be deployed before merging and deploying.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

https://pinwheel.atlassian.net/browse/PP-4622

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const PINWHEEL_MESSAGE_TYPES = {
   PINWHEEL_EVENT: 'PINWHEEL_EVENT',
 };
 
-const LINK_PAGE_URL = 'https://cdn.getpinwheel.com/link-v3.1.0.html';
+const LINK_PAGE_URL = 'https://cdn.getpinwheel.com/link-v3.2.0.html';
```
Comment (roscioli) [inline src/index.tsx:21]: we bumped to 2.2 right?

Comment (TrekSoft) [reply src/index.tsx:21]: Yup this was the piece I mentioned I have to update here

File: src/index.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import {WebView} from 'react-native-webview';
 import {Linking, Platform, SafeAreaView, StyleSheet} from 'react-native';
 
-const version = '2.1.0';
+const version = '2.2.0';
```
Comment (roscioli) [inline src/index.tsx:5]: curious, can we import this from `package.json`?

Comment (TrekSoft) [reply src/index.tsx:5]: We tried but ran into serious issues and there were security reasons mentioned not to do that either when I looked into it.

Comment (roscioli) [reply src/index.tsx:5]: got it!

**Review by roscioli — APPROVED** [review]

File: src/index.tsx (hunk 2 above)
Comment (roscioli) [reply src/index.tsx:5]: may be good to have a unit test testing that this version matches package.json's

Comment (TrekSoft) [reply src/index.tsx:5]: True - I'll add that to my backlog

Comment (roscioli) [reply src/index.tsx:5]: yah, not super urgent but good insurance!

**Pull request merged**